src/prefect/flows.py

```diff
@@ -1061,6 +1068,8 @@ async def deploy(
         entrypoint_type: EntrypointType = EntrypointType.FILE_PATH,
         print_next_steps: bool = True,
         ignore_warnings: bool = False,
+        # Experimental: SLA configuration for the deployment. May be removed or modified at any time. Currently only supported on Prefect Cloud.
+        sla: Optional[Union[SlaTypes, list[SlaTypes]]] = None,
```

Comment:
I think adding it in the docstring is enough
```suggestion
        _sla: Optional[Union[SlaTypes, list[SlaTypes]]] = None,
```